### ISSUE TITLE
bug: combination of number and period

### ISSUE THREAD

**Issue by YohaanYoon** [GetStream/stream-chat-react#1548]
**Describe the bug**

**To Reproduce**

When I enter the combination of number and period in the input box, the message is disappeared.

**Expected behavior**

![chrome-capture-2022-3-27](https://user-images.githubusercontent.com/6019966/165463199-1407b329-20b3-48cc-bd81-c039b3b6aee3.gif)

**Package version**
- stream-chat-react: 8.1.1
- stream-chat-css: 2.9.0
- stream-chat-js: 6.5.0

**Desktop (please complete the following information):**

- OS: mac, windows
- Browser: chrome
- Version: lastest

**Smartphone (please complete the following information):**

**Additional context**

**Comment by YohaanYoon** [comment]
Do you have any plan to fix the bug?
Please let me know when will fix it.

**Comment by petyosi** [comment]
@YohaanYoon I can't seem to replicate the problem in [this basic sandbox](https://codesandbox.io/s/stream-chat-react-tutorial-vq5ntv). Can you provide some more details about your environment? If possible, please fork the setup so that we can replicate the problem on our side.

**Comment by YohaanYoon** [comment]
As you mentioned, the demo app does not reproduce the issue. I will check it.

**Comment by YohaanYoon** [comment]
@petyosi 
The MessageList and MessageInput code is as follows:

```
return (
      <div>
        <S.AppBar />
        <Chat client={chatClient} theme="messaging light" i18nInstance={i18n}>
          <Channel channel={channel}>
            <Window>
              <MessageList
                messageActions={MESSAGE_ACTIONS_CLASSTALK}
              />
              <MessageInput
                doFileUploadRequest={doFileUploadRequest}
                doImageUploadRequest={doFileUploadRequest}
              />
            </Window>
            <Thread />
          </Channel>
        </Chat>
      </div>
    );
```

I have one question. Do you know why combination of number and period is converted to `<li>`  not `<p>`?
Please refer to attached image.

![스크린샷 2022-06-20 오후 2 01 09](https://user-images.githubusercontent.com/6019966/174528597-82654347-6114-4fb3-9f6f-d2d8d4fbccdd.png)

**Comment by petyosi** [comment]
I'm not 100% what you mean, but in general, this can be due to the built-in markdown processing. Check this - https://markdownlivepreview.com/

**Comment by YohaanYoon** [comment]
I will check it. Thanks

**Comment by arnautov-anton** [comment]
Blocked by #1560.

**Comment by arnautov-anton** [comment]
Closing, resolved in #1824.
